doc/varnish-configuration.rst

```diff
@@ -33,6 +33,18 @@ simple ACL, valid for all Varnish versions from 3 onwards, looks as follows:
     trigger invalidation are whitelisted here. Otherwise, lost cache invalidation
     requests will lead to lots of confusion.
 
+Advanced Varnish Configuration: xkey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+With Varnish 4.1 and higher you can optionally take advantage of official Varnish
+modules, specifically `xkey Varnish module`_. This module will provide better
+performance for tag handling as it adds secondary hashes to objects, and as
+it allows for grace handling by means of softpurge on tags.
+
+These modules are bundled with for instance Ubuntu 16.10 and Debian 9
+using package name ``varnish-modules``.
```

Comment:
i'd rather link to the official varnish doc on installation of the modules than give hints for specific distributions (although ubuntu and debian is the real thing, of course ;-) )

what we could mention here is that xkey is not installed alone but all the official modules have to be installed together. (do they also need to be activated, or are they automatically available once installed?)